android_do

```diff
@@ -99,13 +101,18 @@ rm cjdroute
 rm -rf build_linux
 
 export CROSS_COMPILE=$BUILD_PATH/i686-${TYPE}-androideabi/bin/i686-linux-android-
-export TARGET_ARCH=x64
+export PATH=$BUILD_PATH/i686-${TYPE}-androideabi/bin/:$PATH
```

Comment:
Need to be able to find the i686-linux-android-* executables -- although it looks like the absolute path in CROSS_COMPILE should cover that?